Skip test checking output of 'mdrun -h' when patched with PLUMED
Author: Samuel Moors (Vrije Universiteit Brussel)
diff -ur gromacs-2023.3.orig/src/programs/mdrun/tests/CMakeLists.txt gromacs-2023.3/src/programs/mdrun/tests/CMakeLists.txt
--- gromacs-2023.3.orig/src/programs/mdrun/tests/CMakeLists.txt	2023-10-19 10:40:01.000000000 +0200
+++ gromacs-2023.3/src/programs/mdrun/tests/CMakeLists.txt	2024-01-13 12:01:40.289374738 +0100
@@ -49,20 +49,6 @@
 # To avoid running into test timeouts, some end-to-end tests of mdrun
 # functionality are split off. This can be rearranged in future as we
 # see fit.
-set(testname "MdrunOutputTests")
-set(exename "mdrun-output-test")
-
-gmx_add_gtest_executable(${exename}
-    CPP_SOURCE_FILES
-        compressed_x_output.cpp
-        helpwriting.cpp
-        outputfiles.cpp
-        trajectory_writing.cpp
-        # pseudo-library for code for mdrun
-        $<TARGET_OBJECTS:mdrun_objlib>
-    )
-target_link_libraries(${exename} PRIVATE mdrun_test_infrastructure)
-gmx_register_gtest_test(${testname} ${exename} OPENMP_THREADS 2 INTEGRATION_TEST IGNORE_LEAKS QUICK_GPU_TEST)
 
 set(testname "MdrunModulesTests")
 set(exename "mdrun-modules-test")
